docs/operations/count_by_field.md

```diff
@@ -29,19 +28,6 @@ This is an example of counting a field in a record based off a given field
         "standard:0.21.0",
         "kafka:3.2.4"
     ],
-    "apis": [
-        {
-            "_name": "job_metric_api",
-            "default_metrics": false,
-            "port": 3333
-        }
-    ],
-    "external_ports": [
-        {
-            "name": "metrics",
-            "port": 3333
-        }
```

Comment:
I'm assuming this won't be necessary anymore.

Reply:
I think `prom_metrics_port` covers that yeah.

Reply:
Though, we could have forgotten something.

Reply:
Read that ... does the enabling the exporter on a worker (regardless of mechanism) result in the inclusion of the port on the podspec as shown in those docs?

https://terascope.github.io/teraslice/docs/configuration/clustering-k8s#external-ports

Reply:
Oh, then we will need to include `external_ports` on any job that has an exporter enabled, not just `count_by_field` jobs.

Reply:
Well, we shouldn't make the user do it, we should add that as a feature ... if its not there now ... we can live with the manual inclusion of the `external_ports` on jobs.  But it should be automatic if the feature is enabled.  Otherwise the port won't be forwarded from the container.